tracee-ebpf/tracee/events_arm64.go

```diff
@@ -352,7 +352,7 @@ const (
 
 // following syscalls are undefined on arm64
 const (
-	OpenEventID int32 = iota + 10000
+	OpenEventID int32 = iota + Unique32BitSyscallsStartID
```

Comment:
These are not Unique32BitSyscallsStartID, but undefined values.
We should either keep 10000 or set a new macro for it

Reply:
My problem is with this piece of "printList()":

```
for i := tracee.Unique32BitSyscallsStartID; i < int(tracee.Unique32BitSyscallsEndID); i++ {
        index := int32(i)
        event, ok := tracee.EventsIDToEvent[index]
        if !ok {
                continue
        }
        if event.Sets != nil {
                eventSets := fmt.Sprintf("%-22s %-40s %s\n",
                event.Name, fmt.Sprintf("%v", event.Sets),
                fetchFormattedEventParams(index))
                b.WriteString(eventSets)
        } else {
                b.WriteString(event.Name + "\n")
        }
}
```

that only works for x86 currently, right ?

Reply:
I kept 10000 for now and created a Unique32BitSyscallsEndID with the same value of  Unique32BitSyscallsStartID for arm64. That fixes the issue for now, we should address this when "fixing" arm64 support.